Stylesheet/TEI2Ontolex.xsl

```diff
@@ -22,8 +22,6 @@
         xmlns:tei="http://www.tei-c.org/ns/1.0"
         xmlns:skos="http://www.w3.org/2004/02/skos/core#">
 
-    <xsl:variable name="LexiconURI" select="'http://www.mylexica.perso/PLI1906'"/>
-
```

Comment:
This is used for processing the samples from Petit Larousse Illustré. I would rather keep this.

Reply:
Apologies. The tests passed so I thought it to be something unused. :sweat_smile: